hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -87,10 +108,7 @@ public void testMapMetrics() {
         node.getConfig().getMapConfig("hazelcast").getEvictionConfig().setEvictionPolicy(EvictionPolicy.LRU);
         node.getConfig().getMapConfig("hazelcast").setInMemoryFormat(InMemoryFormat.NATIVE);
 
-        stubFor(get(urlPathEqualTo("/ping"))
-                .willReturn(aResponse()
-                        .withStatus(200)));
-
+        stubUrls(200, 400, 400, 400);
```

Comment:
Are you sure about the `400` status code? Can we somehow stub a "connection refused" scenario with wiremock? if we need to return a status code, it should be 404 anyway. But it would be better to cover the network error and the 4xx status code scenarios separately.